### PR TITLE
New dashboard filters: month and document type

### LINKED ISSUE #230: New filter "expenses abroad"
Add a filter on expenses being abroad or not on dashboard.

### PR DESCRIPTION
**Pull request by cuducos**
Fix #230 & #231

### CODE REVIEW

**Comment by jtemporal** [comment]
hey @cuducos any idea why code climate is showing two issues?

File: jarbas/dashboard/admin.py [hunk 1]
```diff
@@ -74,6 +88,40 @@ def queryset(self, request, queryset):
         return queryset.suspicions() if self.value() == 'yes' else queryset
 
 
+class MonthListFilter(SimpleListFilter):
+
+    title = 'mês'
+    parameter_name = 'month'
+    options = ((k, v) for k, v in MONTHS.items())
```
Comment (jtemporal) [inline jarbas/dashboard/admin.py]: This should be something like `options = tuple((k, v) for k, v in MONTHS.items())` so it is displayed on the dashboard. Right now this filter is not being shown ;)

**Comment by cuducos** [comment]
> hey @cuducos any idea why code climate is showing two issues to be fixed?

Repetition in the `admin.py` file. Fixed in 0e6067e (also changed what you've mentioned about generator/tuple) ; )

**Comment by jtemporal** [comment]
>also changed what you've mentioned about generator/tuple

I really liked what you did there ;)

**Pull request merged**